sanic/blueprints.py

```diff
@@ -106,10 +108,18 @@ def chain(nested):
                 else:
                     yield i
 
-        bps = BlueprintGroup(url_prefix=url_prefix)
+        bps = BlueprintGroup(
+            url_prefix=url_prefix,
+            version=version,
+            strict_slashes=strict_slashes,
+        )
         for bp in chain(blueprints):
             if bp.url_prefix is None:
                 bp.url_prefix = ""
+            if bp.version is None and version:
+                bp.version = version
+            if bp.strict_slashes is None:
+                bp.strict_slashes = strict_slashes
```

Comment:
This needs to mirror the `for _attr in ["version", "strict_slashes"]` pattern. More specifically, I am not a fan of the fact that this logic is duplicated. Ideally there is only one way to onboard a blueprint into a group, even if it can be done from either side.

Reply:
Let me merge these two into a common way using the group handlers